### PR TITLE
Update 20.treatments.md

### PR DESCRIPTION
**Pull request by byrdjb**
### Description of the proposed additions or changes
Added the FDA's Emergency Use Authorization for remdesivir and specified the clinical trials it was based upon.

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [x] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.treatments.md [hunk 1]
```diff
@@ -112,6 +112,9 @@ Summarize the state of the antiviral approach.
 
 Remdesivir (GS-5734) was developed by Gilead Sciences to treat Ebola Virus Disease.
 It does not have any FDA-approved use.
+However, on May 1, 2020, the FDA issued an Emergency Use Authorization (EUA) for remdesivir for the treatment of hospitalized COVID-19 patients.
+The EUA was based on information from two clinical trials, NCT04280705 and NCT04292899.
```
Comment (cgreene) [inline content/20.treatments.md:116]:
```suggestion
The EUA was based on information from two clinical trials, NCT04280705 and NCT04292899
```

File: content/20.treatments.md [hunk 2]
```diff
@@ -112,6 +112,9 @@ Summarize the state of the antiviral approach.
...
+[@url:https://clinicaltrials.gov/ct2/show/NCT04280705; @url:https://clinicaltrials.gov/ct2/show/NCT04292899; @url:https://www.fda.gov/media/137564/download]
```
Comment (cgreene) [inline content/20.treatments.md:117]:
```suggestion
[@url:https://clinicaltrials.gov/ct2/show/NCT04280705; @url:https://clinicaltrials.gov/ct2/show/NCT04292899; @url:https://www.fda.gov/media/137564/download].
```

**Comment by byrdjb** [comment]
Thanks, @cgreene. I will try to remember to move that text about Ebola to the background section once these commits have been approved since I don't know if I can do it beforehand.

**Comment by cgreene** [comment]
I'm guessing you didn't mean to close this. I am reopening.

**Comment by cgreene** [comment]
I filed https://github.com/greenelab/covid19-review/issues/294 to track updating the section by a more substantial reorganization.

**Comment by cgreene** [comment]
Ok - this is a quick edit that adds more current information. I'm going to merge it - thanks! I filed the issue to track a reorganization of this section.

**Pull request merged**